muser/README.md

```diff
@@ -0,0 +1,37 @@
+README.md
```

Comment:
Please add a .gitignore file to the root of the project to make sure that no files that shouldn't be committed are added. You can copy/paste:
https://github.com/CUTR-at-USF/travel-behavior-analysis/blob/master/.gitignore

Reply:
I think we're still missing this?